main.js

```diff
@@ -0,0 +1,155 @@
+// I have not yet finished my AI function \\
+// My counter does not reset the minutes after each hour, cant figure out why \\
+// If invalid move clicked game ends, need to figure that out... \\
+
+
+//-----------------------------------------------------------------------------------------------------------------------------
+// Variables to be used throughout code \\
+
+let player1 = document.getElementById('1player');
```

Comment:
This works fine, but remember you can always use **arrays** to hold more than one value in a single variable.